modules/component/types/extractValueTypeFromAction.ts

```diff
@@ -0,0 +1,14 @@
+import {Action} from '@action-land/core'
+
+/**
+ * infers type of values from action set given action type's type
```

Comment:
Need to talk to @tusharmath about this... this sentence is super confusing cuz the word `type` has two meanings. No joke, it's a huge amount of lost productivity sometimes to talk about actions and typings. 😅